azure/v25.0.0/README.md

```diff
@@ -1 +1,16 @@
-# :zap: Giant Swarm Release v25.0.0 for CAPZ :zap:
+# :zap: Giant Swarm Release v25.0.0 for Azure :zap:
+
+We are happy to announce our first release for Cluster API for Azure (CAPZ) that uses the new releases framework.
+
+## Migration to new releases flow
+
+In order to consume the new flow, the following two fields need to be manually adapted:
+
+* In ConfigMap `<cluster name>-userconfig` set `.Values.global.release` to the release version, e.g. `25.0.0`.
+* In App `<cluster name>` set the `version` to an empty string.
```

Comment:
```suggestion
* In App `<cluster name>` remove the `spec.version` field.
* If using GitOps, after pushing the updated manifests, Flux might complain that the App manifest is invalid as the `spec.version` field is mandatory. If that's the case, edit the live App CR and set `spec.version` to an empty string. This will unblock Flux and let it reconcile successfully.
```

Reply:
@iuriaranda can we just default to setting the value to an empty string (how it was) rather than doing the whole dance of trying to remove it first etc? Or do we really want the field gone?